docs/source/optimization_inc.mdx

```diff
@@ -106,6 +106,33 @@ quantizer.quantize(
     calibration_dataset=calibration_dataset,
     save_directory=save_dir,
 )
+
+### Specify Quantization Recipes
```

Comment:
I would prefer to add it in the documentation once the integration is validated on our side

Reply:
will merge this first, and Let's discuss how to improve later